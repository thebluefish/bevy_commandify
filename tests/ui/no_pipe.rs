use bevy_commandify::*;

#[command(pipe=)]
fn foo(world: &mut World) { }

/// Test that a command with a pipe specifies the pipe fn.
fn main() { }