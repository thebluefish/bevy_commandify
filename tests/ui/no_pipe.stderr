error: unexpected end of input, expected an expression
 --> tests/ui/no_pipe.rs:3:1
  |
3 | #[command(pipe=)]
  | ^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `command` (in Nightly builds, run with -Z macro-backtrace for more info)
